packages/embed/src/utils/build-iframe-src.ts

```diff
@@ -42,6 +47,17 @@ export const buildIframeSrc = (params: BuildIframeSrcOptions): string => {
     .forEach(([paramName, paramValue]) => {
       url.searchParams.set(paramName, paramValue)
     })
+  if (embedId) {
+    url.searchParams.set('typeform-embed-id', embedId)
```

Comment:
This is already added in `mapOptionsToQueryParams`, maybe this was just a bad rebase on latest master.